plugins/prow-job/.claude-plugin/plugin.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "prow-job",
+  "description": "A plugin to analyze and inspect Prow CI job results",
+  "version": "0.0.1",
+  "author": {
+    "name": "github.com/openshift-eng"
```

Comment:
can you change the author to your / your  team gh username https://github.com/openshift-eng/ai-helpers/issues/19?